src/styles/utils/variables.scss

```
@@ -0,0 +1,12 @@
+$daySize: 100px;
+$gapSize: 1px;
+
+$days:(
+  "mon": 0,
+  "tue": 1,
+  "wed": 2,
+  "thu": 3,
+  "fri": 4,
+  "sat": 5,
+  "sun": 6,
```

With the `index` approach, you don't have to use these numbers, or rather shouldn't
```suggestion
  "mon",
  "tue",
  "wed",
  "thu",
  "fri",
  "sat",
  "sun",
```